improver/cli/construct_reliability_tables.py

```diff
@@ -11,12 +11,16 @@
 @cli.clizefy
 @cli.with_output
 def process(
-    *cubes: cli.inputcube,
-    truth_attribute,
+    forecast_directory: cli.inputpath,
```

Comment:
As noted by @cpelley, these changes would notably impact other users that use this CLI currently. We would need to communicate this.